src/main/java/nl/stil4m/mollie/domain/CreatePayment.java

```diff
@@ -7,14 +7,19 @@
     private final Double amount;
     private final String description;
     private final String redirectUrl;
+    private final String webhookUrl;
     private final String method;
     private final Map<String, Object> metadata;
 
     public CreatePayment(String method, Double amount, String description, String redirectUrl, Map<String, Object> metadata) {
+        this(method, amount, description, redirectUrl, null, metadata);
+    }
+    public CreatePayment(String method, Double amount, String description, String redirectUrl, String webhookUrl, Map<String, Object> metadata) {
```

Comment:
Please format with an extra enter.

PS: I will deprecate the old constructor in the future